synapse/_scripts/update_synapse_database.py

```diff
@@ -97,10 +96,6 @@ def main() -> None:
     # Load, process and sanity-check the config.
     hs_config = yaml.safe_load(args.database_config)
 
-    if "database" not in hs_config:
-        sys.stderr.write("The configuration file must have a 'database' section.\n")
-        sys.exit(4)
-
```

Comment:
Does this mean we no longer error when both `database` and `databases` are missing?

Also does anything rely on the exit code being 4?

Reply:
I think we do this checking at:

https://github.com/matrix-org/synapse/blob/493c2fc44abcf3457953cc2f6f23509ff7855253/synapse/config/database.py#L97-L123

Maybe I missed something though!